package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "homepage": ".",
+  "name": "react-typescript-starter-pack",
+  "version": "0.1.0",
+  "dependencies": {
+    "@ducanh2912/react-hls-player": "^3.3.5",
+    "@reduxjs/toolkit": "^1.9.3",
+    "@testing-library/jest-dom": "^5.12.0",
+    "@testing-library/react": "^11.2.6",
+    "@testing-library/user-event": "^12.8.3",
+    "@types/classnames": "^2.3.1",
+    "@types/jest": "^26.0.23",
+    "@types/node": "^12.20.10",
+    "@types/react": "^17.0.4",
+    "@types/react-dom": "^17.0.3",
+    "classnames": "^2.3.2",
+    "hls.js": "^1.3.5",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-hls": "0.0.8",
+    "react-hls-player": "^3.0.7",
+    "react-redux": "^8.0.5",
+    "react-router-dom": "^6.9.0",
+    "react-scripts": "4.0.3",
+    "react-video-js-player": "^1.1.1",
+    "redux-persist": "^6.0.0",
+    "typescript": "^4.2.4",
+    "typescript-eslint-parser": "^22.0.0",
```

Comment:
Types, testing libs, eslint plugins all should be in dev dependencies. They are not needed for the production environment